### ISSUE TITLE
[feature]: documentation for self-hosted installation without Docker

### ISSUE THREAD

**Issue by olivierlambert** [makeplane/plane#2145]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

The current guide provides a guide to use Docker to deploy/install. But not everybody use this or trust everything inside various Docker layers.

### Why should this be worked on?

It might be interesting to provide a guide on how to install it "manually" (ie from a git pull) and also how to be updated. An example that is relevant and relatively "close" to this project is Cal.com. See:
* https://cal.com/docs/introduction/quick-start/self-hosting/installation
* https://cal.com/docs/introduction/quick-start/self-hosting/upgrading

You can find some containers around if you need, but giving the explanation on how to install it yourself might also help to get more contributors :) Not everyone uses Docker (for many different reasons).

Obviously, as it's an open source project, if you are interested that I contribute to provide such a guide and update the README accordingly, let me know (but it's maybe better as the original dev you know what to do and what not)

Thanks!

**Comment by henit-chobisa** [comment]
Hey @olivierlambert thank you so much for opening this issue and your valuable insights on this topic. That would really add up a lot to new contributors and for self hosting folks. I convey this internally and find out the best way to do it and get back to you. Your suggestions are welcome. Thanks! ✌🏼

**Comment by pablohashescobar** [comment]
Hello @olivierlambert, our team is currently developing a solution that aims to remove Docker as a requirement for self-hosting Plane. We're excited about this update and will keep you informed on our progress. Stay tuned for more details!

**Comment by olivierlambert** [comment]
Thank you very much! I'm using the quick self-hosted Docker stuff to test and report bugs, which is already pretty handy, until I'll go in a more production ready setup. Thanks a lot!

**Comment by mguptahub** [comment]
@olivierlambert 

We are working on creating the non-docker setup. It will be available soon.

**Comment by olivierlambert** [comment]
Thank you very much! Semi-related question: is it possible to "transfer" the data from a container instance to a non-dockerized one? I already created many issues, so if there's a way to make it "more" painless, I'm taking it. Assuming it's a bit more than just pointing to the new postgresql database after dumping data from the previous one?

**Comment by mguptahub** [comment]
Yes. it is possible to copy the content from docker container to host machine. Simple commands like `docker cp` will help you get what you are looking for.

**Comment by iokarkan** [comment]
Hi! Thanks for the updates. Just to restore my sanity, self-hosting without ./setup.sh will require more than just git checkout the main branch and change an nginx env var, right? Cause I've been trying to deploy by cloning the repo (in a raspberry pi, where non-arm pre-built images won't work) and the app basically doesn't load, may load the god-mode or onboarding screen after many refreshes (if at all), with lots of errors reported in the console, mainly from the api server.

**Comment by kuon** [comment]
We are considering plane, and for many reasons, we are not using docker anymore. I am very happy that you are giving this issue time and attention. I wish you the best.

**Comment by vihar** [comment]
Everything we support today is containerised: Docker Compose, Docker AIO, Docker Swarm, Podman Quadlets, and Kubernetes with Helm — https://developers.plane.so/self-hosting/overview

There's no non-Docker install guide and it isn't on our roadmap.

Closing this out.
